src/components/mainView.tsx

```diff
@@ -1,9 +1,10 @@
-import InputModal from "components/inputModal";
+import StudentInputModal from "components/inputModal/inputModal";
```

Comment:
you can have an `index.tsx` file instead of `inputModal.tsx` to import like `import StudentInputModal from "components/inputModal";`